### PR TITLE
Some minor things

### PR DESCRIPTION
**Pull request by frittentheke**
- Update TF and OpenStack provider requirements
- Use port "0" to set it to "any", otherwise the provider will report constant changes
- Rename the instances names to cp/worker and add an example for nodes required for LFS458

### CODE REVIEW

**Review by maxbischoff — APPROVED** [review]
LGTM (only the re-added template provider lock seems weird to me, but that doesn't seem to be an issue)

File: .terraform.lock.hcl [hunk 1]
```diff
@@ -40,12 +38,28 @@ provider "registry.terraform.io/hashicorp/null" {
...
+    "zh:0e3a6c8e16f17f19010accd0844187d524580d9fdb0731f675ffcf4afba03d16",
+    "zh:45f2c594b6f2f34ea663704cc72048b212fe7d16fb4cfd959365fa997228a776",
+    "zh:77ea3e5a0446784d77114b5e851c970a3dde1e08fa6de38210b8385d7605d451",
+    "zh:8a154388f3708e3df5a69122a23bdfaf760a523788a5081976b3d5616f7d30ae",
+    "zh:992843002f2db5a11e626b3fc23dc0c87ad3729b3b3cff08e32ffb3df97edbde",
+    "zh:ad906f4cebd3ec5e43d5cd6dc8f4c5c9cc3b33d2243c89c5fc18f97f7277b51d",
+    "zh:c979425ddb256511137ecd093e23283234da0154b7fa8b21c2687182d9aea8b2",
+  ]
+}
+
```
[the remark this answers is not in the document]
Comment (frittentheke) [reply .terraform.lock.hcl:61]: My bad ... I has a local state file that still used that ... that's why.

**Comment by frittentheke** [comment]
> LGTM (only the re-added template provider lock seems weird to me, but that doesn't seem to be an issue)

Yeah ... but fixed now ;-)

**Comment by maxbischoff** [comment]
https://github.com/inovex/lfs458-course-prep/blob/master/.github/workflows/terraform.yml#L13 also needs updating

**Review by dirkcjelli — APPROVED** [review]
LGTM

**Pull request merged**